delfin/task_manager/scheduler/schedulers/telemetry/telemetry_job.py

```diff
@@ -0,0 +1,86 @@
+# Copyright 2021 The SODA Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from datetime import datetime
+
+import six
+from oslo_log import log
+from oslo_utils import importutils
+from oslo_utils import uuidutils
+
+from delfin import db
+from delfin.common.constants import TelemetryCollection
+from delfin.task_manager.scheduler import scheduler
+from delfin.task_manager.scheduler.schedulers.telemetry.failed_telemetry_job \
+    import FailedTelemetryJob
+
+LOG = log.getLogger(__name__)
+
+
+class TelemetryJob(object):
```

Comment:
Can we use abstract class here?

Reply:
Yes needed. Added issue to track : https://github.com/sodafoundation/delfin/issues/528